### PR TITLE
Synchronize with BCD v5.2.63

### PR DESCRIPTION
**Pull request by OnkarRuikar**
[BCD release v5.2.63](https://github.com/mdn/browser-compat-data/releases/tag/v5.2.63)

/cc @queengooborg @sideshowbarker

### CODE REVIEW

**Review by dipikabh — APPROVED** [review]
Changes look good to me.

How does this work? For example `scripting` was also updated here - https://github.com/mdn/content/pull/27192 (minus the css/@media change), minus the css/@media changes

**Comment by OnkarRuikar** [comment]
> How does this work? For example `scripting` was also updated here - #27192, minus the css/@media change

These PRs `Synchronize with BCD v...` are run automatically at midnight after BCD repo makes a release. The bot updates content front-mater, headers, and inline badges as per the released BCD. We want BCD as the single source of truth(we had cases where statuses on content didn't get accepted in BCD repo and other people were getting upset over mismatch).\
Your [corresponding PR in BCD repo](https://github.com/mdn/browser-compat-data/pull/20068) didn't make it in BCD `v5.2.62` release last week hence changes in content got reverted. In the latest release your changes made it, so the status is experimental as per current BCD.

In short, it is not required to update statuses in content repo as the bot automatically does it. But if you do then make sure BCD PR makes it in the upcoming BCD release. No harm done if it misses the release, as it gets auto corrected after next BCD release like this.

**Comment by Ontoyou22** [comment]
Stop it or I'm turning you in!
